Src/WitsmlExplorer.Frontend/components/GlobalStyles.tsx

```diff
@@ -97,6 +97,20 @@ const GlobalStyles = createGlobalStyle`
       }
     }
   }
+  .sideBarView  .MuiTreeItem-root {
```

Comment:
Any reason these styles are here instead of the existing SidebarTreeView style on line 54 in Sidebar.tsx?